lib/options_test.go

```diff
@@ -76,6 +78,26 @@ func TestOptionsApply(t *testing.T) {
 		assert.True(t, opts.InsecureSkipTLSVerify.Valid)
 		assert.True(t, opts.InsecureSkipTLSVerify.Bool)
 	})
+	t.Run("TLSCipherSuites", func(t *testing.T) {
```

Comment:
Please test each individual cipher suite as well! It'd suck if we had a couple of suites that just, didn't work…